test/e2e/apps/statefulset.go

```diff
@@ -248,6 +248,124 @@ var _ = SIGDescribe("StatefulSet", func() {
 			sst.WaitForRunningAndReady(*ss.Spec.Replicas, ss)
 		})
 
+		// This can't be Conformance yet because it depends on a default
+		// StorageClass and a dynamic provisioner.
+		It("should perform rolling updates and roll backs of template modifications with PVCs", func() {
```

Comment:
From the test description, I'd have expected that we'd exercise some PVC option explicitly. I think the test name could use revising.

Reply:
Hmm, from the linked issue, does this actually use an RWO PVC?